src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,204 @@
+- `<openshift-cluster-name>`: for example, `okteto`
+
+## Getting your Okteto License
+
+A license is mandatory to use Okteto. You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
+
+## Preparing your cluster for Okteto installation
+
+Before installing Okteto, prepare your Red Hat OpenShift cluster by following these steps.
+
+### Create the `okteto` namespace
+
+Run the following command to create the `okteto` namespace:
+
+```bash
+kubectl create namespace okteto
+```
+
+### Allow containers as `root` in the `okteto` namespace
+
+By default, running containers as `root` is not allowed in Red Hat OpenShift.
+Currently, some components of the Okteto Control Plane require `root` privileges (this is something we will fix soon!).
+To allow `root` containers for the Okteto Control Plane components, run the following commands:
+
+```bash
+oc adm policy add-scc-to-user anyuid -z default -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-buildkit -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-registry -n okteto 
+```
+
+These commands grant the `anyuid` Security Context Constraint (SCC) to the specified service accounts in the `okteto` namespace, allowing them to run containers as `root`.
+
+### Grant access to the "privileged" SCC for the Okteto Build service account
+
+By default, running privileged containers is not allowed in Red Hat OpenShift.
+This restriction affects the [Okteto Build](core/build-service.mdx) service.
+To grant privileged access to the Okteto Build service account, run:
+
+```bash
+ oc adm policy add-scc-to-user privileged -z okteto-buildkit -n okteto
+```
+
+This command grants the `privileged` SCC to the `okteto-buildkit` service account in the `okteto` namespace.
+
+### Grant access to the host to the Okteto Daemon service account
+
+By default, access to the host is restricted by default in Red Hat OpenShift.
+This restriction impacts the he [Okteto Daemon](self-hosted/helm-configuration.mdx#daemonset) service.
+To grant the necessary permissionsto the Okteto Daemon, run:
+
+```bash
+oc adm policy add-scc-to-user hostaccess -z okteto -n okteto
+```
+
+This command grants the `hostaccess` SCC to the `okteto` service account in the `okteto` namespace.
+
+### Copy Red Hat OpenShift wildcard certificate to the `okteto` namespace
+
+OpenShift provides the wildcard certificate for the wildcard domain `*.apps.<openshift-base-domain>`.
+We recommend reusing it when installing Okteto to [expose secure and unique endpoints](self-hosted/helm-configuration.mdx#subdomain) for your development environments.
+
+Copy the certificate to the `okteto` namespace by running:
+
+```yaml
+kubectl get secret <openshift-cluster-name>-primary-cert-bundle-secret -n openshift-ingress -o yaml | sed 's/namespace: openshift-ingress/namespace: okteto/' | kubectl apply -f -
```

Comment:
Here is were a prospect had some issues. For them the secret was called: `RELEASE-NAME-secret`

I would add here maybe something like: _If the secret has a different name just check in that namespace a secret that is mounted as a volume in the route pod_

Reply:
that behavior was due to the prospect having a custom certificate. We could add a note saying ‘unless you’ve generated your own certificate’ or something similar, as I don’t think we can cover all possible scenarios.

Reply:
Sure. But they didn't even know where to look or that they were using a custom one I think. 

Adding that the secret should be there and that it could have different name and the check should be enough for them to find it I think